app/src/main/res/values/styles.xml

```diff
@@ -18,4 +18,68 @@
     <style name="bottomSheetStyleWrapper" parent="Widget.Design.BottomSheet.Modal">
         <item name="android:background">@android:color/transparent</item>
     </style>
+
+    <!--Text Styles-->
+    <style name="HeadingOne">
```

Comment:
I think it would make sense to have all of these in a text_styles file, just so that we don't end up with a massive styles file (will be much easier to navigate)

Reply:
Great idea! I'll move these to their own file